### PR TITLE
Lh/people permissions

### LINKED ISSUE #431: person permissions
add can edit people permission util: department admin and admin
hide edit/delete/lookup on person page based on permission 
move `SearchUsers`, `SearchPeople`, `List` out of people controller 
put `[Authorize(Policy = "AnyRole")]` on new controller, department admin on people controller

### PR DESCRIPTION
**Pull request by laholstege**
closes #431

### CODE REVIEW

**Comment by laholstege** [comment]
@srkirkland wait where do you see the `api/search` ? it should be `people/search` and `peopleAdmin` for the protected stuff (i.e. delete, update)

**Comment by srkirkland** [comment]
@laholstege hmm IDK I don't see api/search anymore -- thought I did before but I guess not.  Now that I look again though you should change back the names from the People controller so `PeopleList` -> `List`, create, search, etc.  A little redundant to do `api/people/listPeople`.

File: Keas.Mvc/ClientApp/util/permissions.ts [hunk 1]
```diff
@@ -20,6 +20,11 @@
         return r.some(a => permissionArray.includes(a));
     }
 
+    public static canEditPeople(r: string[]) {
+        const permissionArray = ['DepartmentalAdmin'];
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/util/permissions.ts]: @laholstege Can you use the Role.Codes? Or does that cause problems?
Role.Codes.DepartmentalAdmin

File: Keas.Mvc/ClientApp/components/People/PersonDetails.tsx [hunk 2]
```diff
@@ -85,10 +97,12 @@ export default class PersonDetails extends React.Component<IProps, {}> {
                         assetInUseUpdated={this.props.inUseUpdated}
                         assetEdited={this.props.edited}
                     />
-                    <HistoryContainer
-                        controller="people"
-                        id={this.props.selectedPersonInfo.person.id}
-                    />
+                    {canEdit && (
+                        <HistoryContainer
+                            controller="people"
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/People/PersonDetails.tsx]: Change this to peopleAdmin and it should be fine.

**Pull request merged**